File: 151051/Krupovich/330 & 340 & 350/dc_kr/discussion/src/main/java/ru/bsuir/DiscussionApplication.java
package ru.bsuir;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.cassandra.repository.config.EnableCassandraRepositories;

@SpringBootApplication
public class DiscussionApplication {

    public static void main(String[] args) {
        SpringApplication.run(DiscussionApplication.class, args);
    }
}

File: 251003/Pankratiev/rest/discussion/src/main/kotlin/bsuir/dc/discussion/DiscussionApplication.kt
package bsuir.dc.discussion

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.retry.annotation.EnableRetry

@SpringBootApplication
@EnableRetry
class DiscussionApplication

fun main(args: Array<String>) {
    runApplication<DiscussionApplication>(*args)
}
